### PR TITLE
Rename buttons and tooltips for adding persons

### PR DESCRIPTION
**Pull request by DavidAndreev**

### CODE REVIEW

File: indico/web/templates/forms/_person_link_widget_base.html [hunk 1]
```diff
@@ -24,12 +24,13 @@
...
-                <span id="add-existing-{{ field.id }}" class="i-button highlight">
-                    {% trans %}Choose{% endtrans %}
+                <span id="add-existing-{{ field.id }}" class="i-button highlight text-color"
+                      title="{% trans %}Search among Indico users{% endtrans %}">
+                    {% trans %}Search{% endtrans %}
                 </span>
                 <span id="add-new-{{ field.id }}" class="i-button"
-                      title="{% trans %}Enter a person manually{% endtrans %}">
-                    {% trans %}Enter{% endtrans %}
+                      title="{% trans %}Enter person information manually{% endtrans %}">
```
Comment (nop33) [inline indico/web/templates/forms/_person_link_widget_base.html]: Shouldn't it be "**person's** information"?

Comment (OmeGak) [inline indico/web/templates/forms/_person_link_widget_base.html]: I'm not sure... but according to Google Search:
`person information` - 3.1B results
`person's information` - 76M results

Comment (ThiefMaster) [inline indico/web/templates/forms/_person_link_widget_base.html]: in any case, with "person's" it would be "*the* person's"

Comment (nop33) [inline indico/web/templates/forms/_person_link_widget_base.html]: Alright then, "person information" it is, forget about the comment :stuck_out_tongue_closed_eyes:

Comment (pferreir) [inline indico/web/templates/forms/_person_link_widget_base.html]: "person" is used as an adjective here. So, "person information" is perfectly OK.

**Pull request merged**